test/save_load_scfres.jl

```diff
@@ -0,0 +1,37 @@
+# This test is based on example examples/collinear_magnetism.jl
+# Import statements are not all included in front to test for 
+# error statements in dispatch
+
+using DFTK
+
+a = 5.42352
+lattice = a / 2 * [[-1  1  1];
+                   [ 1 -1  1];
+                   [ 1  1 -1]]
+Fe = ElementPsp(:Fe, psp=load_psp("hgh/lda/Fe-q8.hgh"))
+atoms = [Fe => [zeros(3)]];
+kgrid = [3, 3, 3]
+Ecut = 15
```

Comment:
This is going to be much to expensive too run on the CI for such a simple thing to test for. Pick something cheaper like good old silicon.